.github/CONTRIBUTING.md

```diff
@@ -32,38 +32,48 @@ $ git push origin master --force
+- Improve the reliability of Elektra, by creating new, and improve existing [tests](/doc/TESTING.md).
+  [Check out the Code Coverage Report](https://doc.libelektra.org/coverage/master/debian-bullseye-full/).
+  [You can also create tests for tutorials](https://github.com/ElektraInitiative/libelektra/tree/master/tests/shell/shell_recorder/tutorial_wrapper)
 
 ## Creating a Pull-Request
 
 If you want to publish your local changes to this project you have to create a new pull-request.
 
 1. Open GitHub and navigate to _your_ libelektra-fork
 2. In the tab _Code_ press the _New Pull-Request_ button and choose a title and fill in the [pull-request template](/.github/PULL_REQUEST_TEMPLATE.md)
-3. Now you should be able to set up the _Comparing changes_ settings. Choose the head repository and base repository. E.g.: if you have some changes in the master branch of your forked repository, select *head repository: yournickname/libelektra and *compare: master* and *base repositroy: ElektraInitiative/libelektra* and *base: master\*
+3. Now you should be able to set up the _Comparing changes_ settings. Choose the head repository and base repository.
+   E.g.: if you have some changes in the master branch of your forked repository, select *head repository: yournickname/libelektra and *compare: master* and *base repository: ElektraInitiative/libelektra* and *base: master\*
 4. Add some information about the changes in the release notes (path of the file: /libelektra/doc/news/\_preparation_next_release.md), skipping this step may cause a rejected pull-request
 5. Do not forget to format your code since the build server fails instantly. Take a look at the [coding guidelines](/doc/CODING.md) for more information.
 6. Commit and push your local changes in git (keep in mind to sync your fork - fetch, rebase & push)
-7. Wait for the code-review
+7. Wait for the code-review or ask others to review
 
 ## Code-Review
 
-After creating the pull-request, your code will be reviewed by other developers. If your pull-request passes the review and the build server accepts your changes, they will be merged into the master branch.
-Get rejected? Don't worry... there are many reasons why the build server could fail. Open your created pull-request in GitHub and scroll to the bottom of the site, there should be a list of all checks - by clicking on the _Details_, you get more information about the checks. However, there are also cases where the websites are down, build server gets stuck or times-out etc., please get also an overview of the [build server](/doc/BUILDSERVER.md) to avoid frustration and long review times.
+After creating the pull-request, your code will be reviewed by other developers.
+If your pull-request passes the review and the build server accepts your changes, they will be merged into the master branch.
+
+Got rejected?
```

Comment:
```suggestion
Automated build doesn't pass?
```

I don't think "rejected" is the best term to use here. IMHO only a human can reject a PR.